ruleset.xml

```diff
@@ -1,4 +0,0 @@
-<?xml version="1.0"?>
-<ruleset name="Kdyby/DoctrineCache">
```

Comment:
určitě jsi tohle chtěl smazat?

Reply:
Tohle tam bylo pravděpodobně že ve starší verzi coding standardu nefungovalo to prolinkování souborů přímo v té knihovně. Teď tam referencuji přímo soubor z vendoru https://github.com/Kdyby/DoctrineCache/pull/20/files/6b2cf4220fcfbc33d035aef4524731cc52efb416#diff-354f30a63fb0907d4ad57269548329e3R43 ať se nemusí každá vyjímka řešit per repository

Reply:
mně právě přišlo lepši, když je to takhle v projektu, protože pak snadno můžeš přidavat vyjimky když je někde potřeba

Reply:
Jasně tady asi záleží na nastavené workflow. 
Za mě to ale nabádá k tomu že obecný problém vyřešíš v jednom balíčku a potom na to stejné narazí někdo na dalších repozitářích a je to kontraproduktivní.
Navíc tady důvod k vyjímkám nevidím, tak nechávat tu soubor kdyby náhodou mi taky nepřijde úplně fajn.

Reply:
> Za mě to ale nabádá k tomu že obecný problém vyřešíš v jednom balíčku a potom na to stejné narazí někdo na dalších repozitářích a je to kontraproduktivní.

Souhlas, ale je to samozřejmě o disciplíně a o tom, jestli maintainer takovou věc připustí :) 

> Navíc tady důvod k vyjímkám nevidím, tak nechávat tu soubor kdyby náhodou mi taky nepřijde úplně fajn.

Tim že tam ten soubor necháš, jak jasně definuješ, že připadna vyjimka se ma vyřešit přesně timto zpusobem a tim padem si usnadňuješ práci, protože všechny balíčky o které se staráš jsou v tom workflow jednotné a nikdo ti nedojde vymyšlet nový způsob jak to udělat.

Reply:
Ok, dohodnuto. Rulesety dávat do balíčku pro lokální modifikace.